### PR TITLE
connect bootstrap & design home page

### PR DESCRIPTION
**Pull request by matan2455**
Using bootstrap files will allow us to create descent UI quickly:

-  implemented staticfiles path to link html with CSS & JS files.
-  created static folder for bootstrap and new css files.
-  designed "Givit" home page.

### CODE REVIEW

File: givitsite/static/home.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  .form-signin input[type="email"] {
+    margin-bottom: -1px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+  .form-signin input[type="password"] {
+    margin-bottom: 10px;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+  }
```
Comment (liorsicd) [inline givitsite/static/home.css:110]: missing new line

File: givitsite/templates/home.html [hunk 2]
```diff
@@ -1,11 +1,84 @@
...
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script>window.jQuery || document.write('<script src="bootstrap/js/jquery-slim.min.js"><\/script>')</script>
+    {% load static %}
+    <script src=" {%static 'bootstrap/js/bootstrap.min.js' %}"></script>
+    {% load static %}
+    <script src="{%static '/bootstrap/js/holder.min.js' %}"></script> 
+  </body>
+
+{%endblock%}
+</html>
```
Comment (liorsicd) [inline givitsite/templates/home.html]: missing newline

File: givitsite/static/home.css (hunk 1 above)
Comment (matan2455) [reply givitsite/static/home.css]: took multiple commits to solve that issue - will squash commits on merge

File: givitsite/templates/home.html (hunk 2 above)
Comment (matan2455) [reply givitsite/templates/home.html]: solved

**Review by ifireball — CHANGES_REQUESTED** [review]
Please squash all the small commits in the PR together. 

Consider separating the general addition of Bootstrap work from the more specific work of adding the home page

**Comment by matan2455** [comment]
Thank you for your comments - we will look into outsourcing our bootstrap after the demo.
for now I'm closing this PR since we merged it's changes in to a different PR that will be posted by shortly by tal, and merging this one might cause unnecessary conflicts (we had to combine the changes last night and couldn't wait to merge and sync with master).

**Pull request closed without merging**